### PR TITLE
Hyperledger Cacti 2024 Q4 Quarterly Report

### PR DESCRIPTION
**Pull request by petermetz**

### CODE REVIEW

File: tac/project-updates/2024/2024-Q4-Hyperledger-Cacti.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+- v2.1.0
+
+# Overall Activity in the Past Quarter
+
+- The vast majority of the community's communication happens on the Discord channel. The mailing list is not used much nowadays.
+- The maintainers try to respond to all queries in a timely manner. On average our response time is likely around a couple of days with the tail end being a couple of weeks.
+  * Note that we don't have hard data on above from Discord, so this is just a personal estimate.
+- We are working on a number of technical initiatives: new features, improvements to the CI speed, artifact signing and improving our OpenSSF score in general.
+- The migration to the Jest test runner from tap is still underway but is about 80% finished now, we are close.
+- One of these mentorship projects involves a major change to the Cacti code base and architecture, namely the integration of the Cactus and Weaver modules into a common and comprehensive set of capabilities for cross-network operations. A first-cut version of these common COPM modules is [close to being merged](https://github.com/hyperledger-cacti/cacti/pull/3546) into the main branch.
```
Comment (tkuhrt) [inline tac/project-updates/2024/2024-Q4-Hyperledger-Cacti.md:45]:
```suggestion
- One of the mentorship projects involves a major change to the Cacti code base and architecture, namely the integration of the Cactus and Weaver modules into a common and comprehensive set of capabilities for cross-network operations. A first-cut version of these common COPM modules is [close to being merged](https://github.com/hyperledger-cacti/cacti/pull/3546) into the main branch.
```

Comment (tkuhrt) [inline tac/project-updates/2024/2024-Q4-Hyperledger-Cacti.md:45]: What does COPM stand for?

Comment (petermetz) [reply tac/project-updates/2024/2024-Q4-Hyperledger-Cacti.md:45]: @VRamakrishna @jenniferlianne

Comment (VRamakrishna) [reply tac/project-updates/2024/2024-Q4-Hyperledger-Cacti.md:45]: COPM == Core OPerators Module. It's a package that represents intgrated functionality from Cactus and Weaver, and along the way should replace older legacy packages. It's being developed by a mentee (Jennifer Bell): https://lf-hyperledger.atlassian.net/wiki/spaces/INTERN/pages/21960881/Cacti+Core+Operators+Module+Project+Plan.

**Review by denyeart — APPROVED** [review]

**Review by jimthematrix — APPROVED** [review]

**Review by tkuhrt — APPROVED** [review]

File: tac/project-updates/2024/2024-Q4-Hyperledger-Cacti.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+# Releases
+
+Issued releases:
+- v2.0.0-rc.4
+- v2.0.0-rc.5
+- v2.0.0-rc.6
+- v2.0.0-rc.7
+- v2.0.0
+
+Caveat: We are still working on the publishing workflows that broke down after our latest org migration.
```
Comment (arsulegai) [inline tac/project-updates/2024/2024-Q4-Hyperledger-Cacti.md:31]: Q: Would this impact existing consumers of cacti libraries?

**Review by arsulegai — APPROVED** [review]

File: tac/project-updates/2024/2024-Q4-Hyperledger-Cacti.md (hunk 2 above)
Comment (petermetz) [reply tac/project-updates/2024/2024-Q4-Hyperledger-Cacti.md:31]: @arsulegai Not on a technical level, no, as-in there should be no breakage from this at all. I can see people being disappointed though when they see the release tags in the git commit log but cannot locate the artifacts on npm (which is a temporary setback we hope to overcome soon)

**Pull request merged**